engine/src/field.cpp

```diff
@@ -3205,11 +3207,28 @@ IO_stat MCField::extendedsave(MCObjectOutputStream& p_stream, uint4 p_part)
         t_size += sizeof(uint8_t);
     }
     
+    // SN-2015-04-30: [[ Bug 15175 ]] Save the tabalign property of the field
+    if (ntabs != 0)
+    {
+        t_flags |= FIELD_EXTRA_TABALIGN;
+        // Save number of tab alignments, and then each of them.
+        t_size += sizeof(uint16_t);
+        t_size += sizeof(intenum_t) * nalignments;
```

Comment:
There aren't enough alignments to justify using an intenum_t - the simplest thing to do is to serialize each alignment as bytes (although even that's a little overkill).

(The in-memory rep is abit overkill too!)